vignettes/funbiogeo.Rmd

```diff
@@ -20,405 +21,296 @@ knitr::opts_chunk$set(
+and trait data from the
+[PanTHERIA database](https://esapubs.org/archive/ecol/E090/184/)
+[@Jones_PanTHERIA_2009].
+The initial extraction is available
+[on GitHub](https://github.com/FRBCesab/eumammals).
 
-The package `funbiogeo` requires three different datasets to proceed with the 
+The package `funbiogeo` requires three different data.frame to proceed with the 
 analyses:
 
-- the **site x locations** dataset (example dataset: `site_locations`)
-- the **site x species** dataset (example dataset: `site_species`)
-- the **species x traits** dataset (example dataset: `species_traits`)
-
-Let's take a look at each of these datasets.
+- the **species x traits** data.frame (`species_traits` in `funbiogeo`)
+- the **site x species** data.frame (`site_species` in `funbiogeo`)
+- the **site x locations** object (`site_locations` in `funbiogeo`)
 
+You can load the example data using `data(..., package = "funbiogeo")` call:
 
-### Site x Locations
+```{r load-datasets}
+data("site_species", package = "funbiogeo")
+data("site_locations", package = "funbiogeo")
+data("species_traits", package = "funbiogeo")
+```
 
 
+### Species x Traits
 
-This dataset contains spatial coordinates of sites (points) of the study area. 
-It can be either a `matrix` or a `data.frame` and row names must contain sites 
-names. It must contain at least two columns: the longitude and the latitude of 
-sites. This dataset must contain unique sites. Additional columns are accepted 
-must will be ignored by `funbiogeo`.
+This object contains traits values for studied species.
+It should be a `data.frame`. The first column should contain species names and
+the other columns contain trait values.
 
-The package `funbiogeo` comes with the example dataset `site_locations`.
+Note that we'll be talking about **species** throughout this vignette
+and in the arguments of `funbiogeo`, but the package doesn't make any assumption
+on the biological level at which species are. It can be individuals,
+populations, strains, species, genera, families, etc. The important fact is that
+you should have trait data for the level at which you want to work.
 
+Let's examine the `species_traits` data include in the package:
 
-```{r 'load-sites-locs'}
-## Load site x locations example dataset ----
-data("site_locations", package = "funbiogeo")
+```{r preview-species-traits, echo = FALSE}
+knitr::kable(head(species_traits, 4), 
+             caption   = "First lines of species x traits dataset", 
+             align     = c("r", "r", "r", "r", "r", "r"))
 ```
 
-```{r 'preview-sites-locs', echo = FALSE}
-knitr::kable(head(site_locations, 10), 
-             caption   = "Format of site x locations dataset", 
-             align     = c("r", "r"))
-```
+The first column `species` contains species names, while the next 6 columns
+contains the trait values for all species. Note that the species names are 
+anonymized because the IUCN data
```

Comment:
I didn't receive any answer about sharing anonymized data... So let's say it's OK.